src/NUnitTestAdapterTests/NavigationDataTests.cs

```diff
@@ -47,6 +47,7 @@ public void TearDown()
         [TestCase("+GenericFixture`2", "Matches", 116, 117, 117, 118)]
         [TestCase("+GenericFixture`2+DoublyNested", "WriteBoth", 132, 133, 134, 134)]
         [TestCase("+GenericFixture`2+DoublyNested`1", "WriteAllThree", 151, 152, 153, 153)]
+        [TestCase("+DerivedClass", "EmptyMethod_ThreeLines", 160, 161, 161, 161)]
```

Comment:
What am I missing here? In the code in `NavigationTestData`, `DerivedClass` does not derive from `BaseClass`. How is `EmptyMethod_ThreeLines` being found, especially since it is named differently?